website/server/middlewares/cron.js

```diff
@@ -6,9 +6,10 @@ import Bluebird from 'bluebird';
 import { model as Group } from '../models/group';
 import { model as User } from '../models/user';
 import { recoverCron, cron } from '../libs/cron';
-import { v4 as uuid } from 'uuid';
 
 const daysSince = common.daysSince;
+// Wait this length of time in ms before attempting another cron
+const CRON_TIMEOUT = new Date(60 * 60 * 1000).getTime();
```

Comment:
Gonna go with CRON_TIMEOUT_WAIT if nobody has a better suggestion, it seems most accurate in my mind to what the constant is actually storing